File: Api/UserManagement/UserManagement/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace UserManagement.Models
{
    public class UserModel
    {
        public string ID { get; set; } 
        public string UserName { get; set; }
        public Nullable<System.DateTime> BirthDate { get; set; }
        public string BirthDateConvert { get; set; }
        public string GendelName { get; set; }
        public Nullable<System.DateTime> CreatedDate { get; set; }
        public Nullable<System.DateTime> DeletedDate { get; set; }
        public Nullable<System.DateTime> UpdatedDate { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public Nullable<decimal> IsDelete { get; set; }
        public Nullable<decimal> Gendel { get; set; }
    }
}

File: Client/UserManagerment/src/app/shared/user.model.ts
 
export class User {
    ID: string;
    UserName: string;
    PhoneNumber: string;
    Email: string;
    Password: string;
    IsDelete: number;
    Gendel: number;
    BirthDate: string;
    BirthDateConvert: string;
    CreatedDate: string;
    DeletedDate: string;
    UpdatedDate: string;
  constructor() {
      this.ID = "";
      this.UserName = "";
      this.PhoneNumber = "";
      this.Email = "";
      this.Password = "";
      this.IsDelete = 0;
      this.Gendel = 0;
      this.BirthDate = "";
      this.BirthDateConvert = "";
      this.CreatedDate = "";
      this.DeletedDate = "";
      this.UpdatedDate = "";

    }
  }